classes/class-wc-connect-shipping-label.php

```diff
@@ -38,8 +38,10 @@ protected function get_form_data( WC_Order $order ) {
 			}
 			$formData[ 'cart' ] = $contents;
 
+			foreach( $this->settings_store->get_origin_address() as $key => $value ) {
+				$formData[ 'orig_' . $key ] = $value;
+			}
```

Comment:
This would be a place where you could pre-fill a bunch of stuff ( https://github.com/Automattic/woocommerce-connect-client/pull/442#issuecomment-231435906 )